File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        int n = nums.length;
        int cnt = 0;
        for(int a = 0;a < n;a++){
            for(int b = a+1;b<n;b++){
                if(nums[a]==nums[b]) cnt++;
            }
        }if(cnt>0){
            return true;
        }
        return false;
    }
}

File: Data Structures & Algorithms/search-in-rotated-sorted-array-ii/submission-1.cpp
class Solution {
public:
    int search(vector<int>& nums, int target) {
        int low=0, high=nums.size()-1;
        while(low<=high){
            int mid = low + (high-low)/2;
            if(nums[mid] == target) return 1;
            
            // DUPLICATES
            if(nums[low] == nums[mid] && nums[mid] == nums[high]){
                low++;
                high--;
                continue;
            }

            // LEFT SIDE SORTED
            if(nums[low] <= nums[mid]){
                if(nums[low] <= target && target <= nums[mid]) high = mid-1;
                else low = mid+1;
            }

            // RIGHT SIDE SORTED
            else{
                if(nums[mid] <= target && target <= nums[high]) low = mid+1;
                else high = mid-1;
            }
        }
        return 0;
    }
};